lib/client-js/campus.js

```diff
@@ -153,6 +153,14 @@ var campus = {
                         {
                           name: "einsteins",
                           id: "einsteinbrosbagels"
+                        },
```

Comment:
Thinking about this change more, I would say a better solution would have been to use a regex search. This does get the job done, just a little messier.